includes/vendor/myparcelnl/sdk/src/Helper/ValidateStreet.php

```diff
@@ -62,15 +62,12 @@ public static function getStreetRegexByCountry(string $local, string $destinatio
     {
         if (
             ($local === AbstractConsignment::CC_NL && $destination === AbstractConsignment::CC_NL) ||
-            ($local === AbstractConsignment::CC_NL && $destination === AbstractConsignment::CC_BE)
+            ($local === AbstractConsignment::CC_BE && $destination === AbstractConsignment::CC_NL)
         ) {
             return self::SPLIT_STREET_REGEX_NL;
         }
 
-        if (
-            ($local === AbstractConsignment::CC_BE && $destination === AbstractConsignment::CC_BE) ||
-            ($local === AbstractConsignment::CC_BE && $destination === AbstractConsignment::CC_NL)
-        ) {
+        if ($local === AbstractConsignment::CC_BE && $destination === AbstractConsignment::CC_BE) {
```

Comment:
Het maakt toch niet uit wat de local cc is? Als de destination BE is moet er een boxnummer in het adres kunnen

Reply:
smort